docs/api/effector/babel-plugin.md

```diff
@@ -88,150 +88,151 @@ getUsers.use(
-effector 20.2.0
+effector 21.6.0
 :::
 
-Option for effector/babel-plugin for making custom unit factories with clean configuration.
+- Type: `string[]`
+
+Accepts an array of module names which exports treat as custom factories therefore each function call provides unique prefix for [sids](./babel-plugin.md#sid) of units inside them. Used to SSR([Server Side Rendering](https://developers.google.com/web/updates/2019/02/rendering-on-the-web)) and it's not required for client-only application.
+
+- Factories can have any amount of arguments.
+- Factories can create any amount of units.
+- Factories can call any effector methods.
+- Factories can call another factories from others modules.
+- Modules with factories can export any amount of functions.
+- Factories should be compiled with `effector/babel-plugin` as well as code which use them.
+
+#### Example
 
 ```json title=".babelrc"
 {
   "plugins": [
-    ["effector/babel-plugin", {"addLoc": true}],
     [
       "effector/babel-plugin",
       {
-        "importName": "@lib/createInputField",
-        "storeCreators": ["createInputField"],
-        "noDefaults": true
-      },
-      "createInputField"
+        "factories": ["src/createEffectStatus", "~/createCommonPending"]
+      }
     ]
   ]
 }
 ```
 
-```js title="@lib/createInputField.js"
-import {createStore} from 'effector'
-import {resetForm} from './form'
+```js title="./src/createEffectStatus.js"
+import {rootDomain} from './rootDomain'
 
-export function createInputField(defaultState, {sid, name}) {
-  return createStore(defaultState, {sid, name}).reset(resetForm)
+export function createEffectStatus(fx) {
+  const $status = rootDomain
+    .createStore('init')
+    .on(fx.finally, (_, {status}) => status)
+  return $status
 }
 ```
 
-```js title="src/state.js"
-import {createInputField} from '@lib/createInputField'
-
-const foo = createInputField('-')
-/*
+```js title="./src/statuses.js"
+import {createEffectStatus} from './createEffectStatus'
+import {fetchUserFx, fetchFriendsFx} from './api'
 
-will be treated as store creator and compiled to
+export const $fetchUserStatus = createEffectStatus(fetchUserFx)
+export const $fetchFriendsStatus = createEffectStatus(fetchFriendsFx)
+```
 
-const foo = createInputField('-', {
-  name: 'foo',
-  sid: 'z&si65'
-})
+Import `createEffectStatus` from `'./createEffectStatus'` was treated as factory function so each store created by it has its own [sid](./babel-plugin.md#sid) and will be handled by [serialize](./serialize.md) independently, although without `factories` they will share the same `sid`.
 
-*/
-```
+### reactSsr
 
-### importName
+:::note since
+effector 21.5.0
+:::
 
-> `importName: string | string[]`
+- Type: `boolean`
+- Default: `false`
 
-Specify import to process by plugin. By default has value `["effector", "effector/compat"]`
+Replace imports from `effector-react` to `effector-react/ssr`. Useful for building both server-side and client-side builds from the same codebase.
```

Comment:
As I know, `effector-react/ssr` should be replaced with `effector-react/scope` after release 22.